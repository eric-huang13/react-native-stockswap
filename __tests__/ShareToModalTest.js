import React from 'react';
import ShareToModal from '../src/components/HomeTabComponents/ShareToModal';
import ShallowRenderer from 'react-test-renderer/shallow'; // ES6


jest.mock("@react-native-async-storage/async-storage", () =>
  require("@react-native-async-storage/async-storage/jest/async-storage-mock"),
);


  test('renders correctly', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<ShareToModal />);
    const tree = renderer.getRenderOutput();
    expect(tree).toMatchSnapshot();
  });