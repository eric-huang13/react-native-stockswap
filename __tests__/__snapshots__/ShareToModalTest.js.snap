// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#3e4d6c",
      "borderRadius": 28.248175182481752,
      "elevation": 5,
      "flex": 1,
      "marginTop": 564.963503649635,
      "padding": 19.77372262773723,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.25,
      "shadowRadius": 4,
    }
  }
>
  <ForwardRef
    onPress={[Function]}
  >
    <Text
      style={
        Object {
          "color": "#B8A0FF",
          "fontFamily": "Montserrat-Bold",
          "fontSize": 19.77372262773723,
          "marginBottom": 25.423357664233578,
        }
      }
    >
      Close
    </Text>
  </ForwardRef>
  <Text
    style={
      Object {
        "color": "white",
        "fontFamily": "Montserrat-Medium",
        "fontSize": 19.77372262773723,
        "marginHorizontal": 16.94890510948905,
        "paddingVertical": 2.8248175182481754,
      }
    }
  >
    Share to...
  </Text>
  <View
    style={
      Object {
        "flexDirection": "column",
        "height": "50%",
        "justifyContent": "space-between",
        "marginTop": 42.372262773722625,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontFamily": "Montserrat-Medium",
          "fontSize": 16.94890510948905,
          "marginHorizontal": 16.94890510948905,
          "paddingVertical": 2.8248175182481754,
        }
      }
    >
      Option 1
    </Text>
    <Text
      style={
        Object {
          "color": "white",
          "fontFamily": "Montserrat-Medium",
          "fontSize": 16.94890510948905,
          "marginHorizontal": 16.94890510948905,
          "paddingVertical": 2.8248175182481754,
        }
      }
    >
      Option 2
    </Text>
    <Text
      style={
        Object {
          "color": "white",
          "fontFamily": "Montserrat-Medium",
          "fontSize": 16.94890510948905,
          "marginHorizontal": 16.94890510948905,
          "paddingVertical": 2.8248175182481754,
        }
      }
    >
      Option 3
    </Text>
  </View>
</View>
`;
